xla/tests/BUILD

```diff
@@ -2378,6 +2378,7 @@ xla_cc_test(
         "local_client_aot_test.cc",
         ":local_client_aot_test_computation.o",
     ],
+    tags = ["xla_disable"],
```

Comment:
What does this tag do?

Reply:
I had this in by accident from debugging, removed it